warehouse/routes.py

```diff
@@ -538,14 +538,18 @@ def includeme(config):
         traverse="/{name}/",
         domain=warehouse,
     )
-    # Integration URLs
 
+    # Integration URLs
+    config.add_route(
+        "integrations.secrets.disclose-token",
+        "/_/secrets/disclose-token",
+        domain=warehouse,
+    )
     config.add_route(
-        "integrations.github.disclose-token",
+        "integrations.github.disclose-token",  # For backwards compatiblity
```

Comment:
does this _need_ a different name/view_config?

Reply:
Yes, I don't think we can mount the same view twice at two different routes.